pinot-controller/src/main/resources/app/utils/PinotMethodUtils.ts

```diff
@@ -436,7 +434,7 @@ const getAllTableDetails = (tablesList) => {
   if (tablesList.length) {
     const promiseArr = [];
     tablesList.map((name) => {
-      promiseArr.push(getTableSize(name));
+      promiseArr.push(getTableSize(name, false));
```

Comment:
don't have to pass false if we have default value

Reply:
This default value was left to true just so not to effect the existing flow but I can change the default to false though

Reply:
Done